flight/drone/drone_controller.py

```diff
@@ -140,7 +140,7 @@ def run(self):
             sleep(c.DELAY_INTERVAL)  # Sleep in case was doing write operation
             self._splitter.exit()
 
-    def add_hover_task(self, altitude, duration, priority=c.Priorities.LOW):
+    def add_hover_task(self, duration=c.DEFAULT_HOVER_DURATION, altitude=None, priority=c.Priorities.LOW):
```

Comment:
Just default altitude to be rangefinder distance, since that's the only checking you are doing right now